packages/react-native-bridge/index.js

```diff
@@ -24,6 +24,10 @@ export const userEvents = {
 	editorSessionTemplatePreview: 'editor_session_template_preview',
 };
 
+export const actionButtons = {
+	missingBlockAlertActionButton: 'missing_block_alert_action_button',
```

Comment:
I'm not sure if `missingBlockAlertActionButton` describes well, would it be better to replace "missing" with "unsupported"?

Reply:
The block that is shown for an unsupported block is called `missing`. That's why the naming.

Reply:
I see your point